docs/book/src/clusterctl/commands/upgrade.md

```diff
@@ -123,3 +123,40 @@ clusterctl upgrade apply \
 In this case, all the provider's versions must be explicitly stated.
 
 </aside>
+
+<aside class="note warning">
+
+<h1> Upgrading to Cluster API core components pre-release versions </h1>
+
+Use `clusterctl` CLI options to target the [desired version](https://github.com/kubernetes-sigs/cluster-api/releases).  
+
+The following shows an example of upgrading `bootrap`, `kubeadm` and `core` components to version `v1.6.0-rc.1`:
+
+```bash
+TARGET_VERSION=v1.6.0-rc.1
+
+clusterctl upgrade apply \
+    --bootstrap=kubeadm:${TARGET_VERSION} \
+    --control-plane=kubeadm:${TARGET_VERSION} \
+    --core=cluster-api:${TARGET_VERSION}
+```
+
+</aside>
+
+<aside class="note warning">
+
+<h1> Deploying nightly release images </h1>
```

Comment:
Thank you very much for writing this up!

Can you please try if it's possible to overwrite only the URL for the providers instead of the images (see: https://github.com/kubernetes-sigs/cluster-api/issues/9604#issuecomment-1858153553)

This way we would also get the latest manifests if I understand it correctly

Reply:
👋 @sbueringer have not gotten any traction on the linked issue re adding googlestorage repo support  

wdyt if i update this current PR to remove the example for overwriting the image tag and simplify the doc update to mention the location of the nightly release YAMLs.  

i will also create a follow up issue to address `Add support for consuming nightly release manifests in clusterctl`?

Reply:
Sounds good!

Reply:
pushed up some changes here when you have a chance to take another pass

Reply:
Thx!